types/src/contract_event.rs

```diff
@@ -180,6 +187,9 @@ pub struct ContractEventV1 {
     /// The data payload of the event
     #[serde(with = "serde_bytes")]
     event_data: Vec<u8>,
+    /// Cached event size used for gas charging.
+    #[serde(skip_serializing)]
+    size: usize,
```

Comment:
hmm.. then it'll be zero coming from db? that doesn't sound ideal, maybe use a `Option<usize>`?

Reply:
Ah good catch ://

Reply:
Let me fix this

Reply:
Fixed, just skipped deserialization and added a custom recomputation for size field. Added a test which works now. Thanks!